packages/contracts/development/contracts/RealityETH-4.0.sol

```diff
@@ -264,386 +115,16 @@ contract RealityETH_v4_0 is BalanceHolder, IRealityETHCore, IRealityETHHistoryVe
-    /// @param question_id The ID of the question
-    function getContentHash(bytes32 question_id) public view returns (bytes32) {
-        return questions[question_id].content_hash;
-    }
-
-    /// @notice Returns the arbitrator address for the question
-    /// @param question_id The ID of the question
-    function getArbitrator(bytes32 question_id) public view returns (address) {
-        return questions[question_id].arbitrator;
-    }
-
-    /// @notice Returns the timestamp when the question can first be answered
-    /// @param question_id The ID of the question
-    function getOpeningTS(bytes32 question_id) public view returns (uint32) {
-        return questions[question_id].opening_ts;
-    }
-
-    /// @notice Returns the timeout in seconds used after each answer
-    /// @param question_id The ID of the question
-    function getTimeout(bytes32 question_id) public view returns (uint32) {
-        return questions[question_id].timeout;
-    }
-
-    /// @notice Returns the timestamp at which the question will be/was finalized
-    /// @param question_id The ID of the question
-    function getFinalizeTS(bytes32 question_id) public view returns (uint32) {
-        return questions[question_id].finalize_ts;
-    }
-
-    /// @notice Returns whether the question is pending arbitration
-    /// @param question_id The ID of the question
-    function isPendingArbitration(bytes32 question_id) public view returns (bool) {
-        return questions[question_id].is_pending_arbitration;
-    }
-
-    /// @notice Returns the current total unclaimed bounty
-    /// @dev Set back to zero once the bounty has been claimed
-    /// @param question_id The ID of the question
-    function getBounty(bytes32 question_id) public view returns (uint256) {
-        return questions[question_id].bounty;
-    }
-
-    /// @notice Returns the current best answer
-    /// @param question_id The ID of the question
-    function getBestAnswer(bytes32 question_id) public view returns (bytes32) {
-        return questions[question_id].best_answer;
-    }
-
-    /// @notice Returns the history hash of the question
-    /// @param question_id The ID of the question
-    /// @dev Updated on each answer, then rewound as each is claimed
-    function getHistoryHash(bytes32 question_id) public view returns (bytes32) {
-        return questions[question_id].history_hash;
-    }
-
-    /// @notice Returns the highest bond posted so far for a question
-    /// @param question_id The ID of the question
-    function getBond(bytes32 question_id) public view returns (uint256) {
-        return questions[question_id].bond;
-    }
-
-    /// @notice Returns the minimum bond that can answer the question
-    /// @param question_id The ID of the question
-    function getMinBond(bytes32 question_id) public view returns (uint256) {
-        return questions[question_id].min_bond;
+    function withdraw() public override notFrozen {
```

Comment:
When we do the forking we split all the funds in one go. 

If we allowed withdrawals against the parent contract, we would have to instead keep track of which funds belonged to the parent and which had to be transferred to the child. That would mean that we'd have to either

a) do splitting every time we transferred a question or a balance. 
b) Add extra logic to keep track of the total withdrawable funds so that we could avoid transferring that hold that part back when we transferred the rest to the child.

Much simpler just to just send all the funds to the children and deal with it there.